docs/whats-new/production-release-notes.md

```diff
@@ -2,6 +2,61 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Production release 22 March 2023
+
+Frontend version: 1.0.0-20230315.0325 | Backend version: 1.54.0-230309.0902
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Tenants receive a daily notification when an invoked webhook endpoint fails.
+
+</summary>
+
+Tenants will now receive a daily notification when an invoked webhook endpoint fails. The email informs you of the webhook ID, target URL and webhook event type. You have to check the target URL that has failed and fix the endpoints.
+
+</details>
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Company field is now auto-populated based on vendor's email domain when inviting user to TechPass</summary>
```

Comment:
this is only in stg & not available in Prod yet, can remove from prod release notes

Reply:
in the staging release note was it indicated that the change will not be promoted to prod? if not we should add it to manage expectation

Reply:
removed

Reply:
updated the staging with the note Jerome suggested.